app/views/profile/_profile.html.erb

```diff
@@ -0,0 +1,66 @@
+<section class="p-5">
+  <h1 class="text-2xl font-semibold text-[#202020] mb-1">
+    Mentor Profile
+  </h1>
+  <p class="text-sm text-[#898989] mb-5">Discover insights and stories from our mentors and mentees to inspire your own
+    journey.</p>
+
+  <!--top profile card-->
+  <div class="bg-[#FFFFFF] flex gap-5 py-5 border-slate-300 rounded-lg px-5 w-895px h-171px mb-3">
+    <div>
+      <!--    <img src=<%#= user.profile_picture %> alt="user_profile_picture" class="w-24 h-24 object-cover rounded-lg border">-->
+      <img src=<%= asset_path('sample_img/mentor_sample.png') %> alt="user_profile_picture" class="w-24 h-24 object-cover rounded-lg border">
+    </div>
+    <div>
+      <b class="text-lg text-[#202020] font-bold"><%= user.first_name %> <%= user.last_name %></b>
+      <p class="text-sm	text-[#202020] mb-2.5"><%= user.title %></p>
+      <div>
+        <div class="flex gap-1 mb-2">
+          <img src="<%= asset_path('icon/office.svg') %>" alt="office" class="max-w-15 max-h-15">
+          <p class="text-xs text-[#202020]">Discovery Partners Institute</p>
+        </div>
+        <!--      <p><%#= user.company %></p> TODO-->
```

Comment:
Are you able to:
-Put "TODO:" first? 
-Remove spacing in the beginning (happens on a few different lines)

Reply:
Updated :)